### DISCUSSION TITLE
灌水

### DISCUSSION THREAD

**Discussion by 1451374815** [share121/inter-knot#337] [其他]
16号要去原神FES，现在超级紧张，感觉很社恐

**Comment by dongzhiaidao** [comment]
别怕，逮着cos大佬一顿合拍就是

**Reply by 1451374815** [reply]
害怕

**Comment by hitting1234** [comment]
集到邮，要和绳网大伙分享哦😍

**Comment by ShangHuaWenYuan** [comment]
[点❤我❤看❤可❤爱❤伊❤埃❤斯](https://www.bilibili.com/video/BV18E4m1d7fE/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Reply by fu-fu404** [reply]
哥们你有点逆天了

**Comment by Kamisato100** [comment]
平等的羡慕每一个能去FES玩的人

**Comment by liuyingluoyue** [comment]
我我我，我可以和你一块去

**Reply by 1451374815** [reply]
你几号啊

**Reply by 1451374815** [reply]
你好

**Reply by liuyingluoyue** [reply]
我也16号

**Reply by 1451374815** [reply]
好巧，现场见

**Reply by 1451374815** [reply]
可惜这个网站很不稳定，而且只有在电脑上才能用，不然当天都可以用绳网联系了，哈哈

**Comment by Changes1794** [comment]
别怕，以一个半社恐去BW的经验告诉你，coser们都很友善的，遇到就上去集邮就可以了。

**Reply by 1451374815** [reply]
这对一个社恐来说真的很艰难

**Comment by Tsukinoamane** [comment]
没关系，我替你去就不紧张了ヾ(*ΦωΦ)ﾉ

**Reply by 1451374815** [reply]
哈哈哈
